internal/action/main.go

```diff
@@ -161,6 +163,21 @@ func Run(config *Inputs) error {
 		return fmt.Errorf("failed to parse backend configuration: %s", err)
 	}
 
+	var tagInputs Tags
+	if err = yaml.Unmarshal([]byte(config.Tags), &tagInputs); err != nil {
+		return fmt.Errorf("failed to decode tag names: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to decode tag names: %w", err)
```

Errors should be wrapped, which is required to use `errors.Is` and `errors.As`